### PR TITLE
aruha-730: use elastic ip address for exhibitors

### PR DESCRIPTION
**Pull request by adyach**
This PR brings possibility to use elastic ips for exhibitor instances in order to avoid restarting Kafka on exhibitor ips list change. The public ip is not available for the whole internet.

### CODE REVIEW

File: bubuku/env_provider.py [hunk 1]
```diff
@@ -63,11 +63,11 @@ def _load_instance_ips(self, lb_name: str):
 
         for instance in response['InstanceStates']:
             if instance['State'] == 'InService':
-                private_ips.append(ec2.describe_instances(
-                    InstanceIds=[instance['InstanceId']])['Reservations'][0]['Instances'][0]['PrivateIpAddress'])
+                public_ips.append(ec2.describe_instances(
+                    InstanceIds=[instance['InstanceId']])['Reservations'][0]['Instances'][0]['PublicIpAddress'])
```
Comment (antban) [inline bubuku/env_provider.py]: Probably it is better to somehow determine situation when public is not specified for one or several instances

Comment (adyach) [inline bubuku/env_provider.py]: 1b0ed02

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
There is no need in it, since it is not possible to use elastic ips without opening exhib to the world.

**Pull request closed without merging**